src/QMCWaveFunctions/detail/CUDA/cuBLAS_LU.cu

```diff
@@ -0,0 +1,253 @@
+                   "cuBLAS::getri_batched failed in computeInverseAndDetLog_batched");
+}
+
+
+void computeGetri_batched(cublasHandle_t& h_cublas,
+                          const int n,
+                          const int lda,
+                          double* Ms[],
+                          double* Cs[],
+                          int* pivots,
+                          int* infos,
+                          const int batch_size)
+{
+  cublasErrorCheck(cuBLAS::getri_batched(h_cublas, n, Ms, lda, pivots, Cs, lda, infos, batch_size),
+                   "cuBLAS::getri_batched failed in computeInverseAndDetLog_batched");
+}
+
+template void computeGetrf_batched<double>(cublasHandle_t& h_cublas,
+                                           cudaStream_t& hstream,
+                                           const int n,
+                                           const int lda,
+                                           double* Ms[],
+                                           int* pivots,
+                                           int* host_infos,
+                                           int* infos,
+                                           const int batch_size);
+
+template void computeGetrf_batched<std::complex<double>>(cublasHandle_t& h_cublas,
+                                                         cudaStream_t& hstream,
+                                                         const int n,
+                                                         const int lda,
+                                                         std::complex<double>* Ms[],
+                                                         int* pivots,
+                                                         int* host_infos,
+                                                         int* infos,
+                                                         const int batch_size);
+
+
+template void computeLogDet_batched<std::complex<double>>(cudaStream_t& hstream,
+                                                          const int n,
+                                                          const int lda,
+                                                          std::complex<double>** LU_mat,
+                                                          const int* pivots,
+                                                          std::complex<double>* log_dets,
+                                                          const int batch_size);
+
+template void computeLogDet_batched<double>(cudaStream_t& hstream,
+                                            const int n,
+                                            const int lda,
+                                            double** LU_mat,
+                                            const int* pivots,
+                                            std::complex<double>* log_dets,
+                                            const int batch_size);
+
+template void computeLogDet_batched<float>(cudaStream_t& hstream,
```

Comment:
Is float used? if not, please remove.

Reply:
Maybe in mixed precision, enough has changed here that I want to reserve that option for the next two PR's